network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -309,6 +309,7 @@ def categories_page(self, request, slug):
 
         context['category'] = slug
         context['current_category'] = category
+        context['pagetype'] = 'category_page'
```

Comment:
Can you add some context why this needs to be available in the context?

Reply:
if you go to `/categories/<any of them>` on load when you do not add that specific line, the `All Reviews` link will be in an active state as well as the category being highlighted. We needed a way to indicate that a category link is not the home page in the case

Reply:
@fessehaye I guess this is related to my other question then. What do you think about making this flag express what the intended effect is rather than having this page type flag where it is not visible from the name what it is used for?